index.html

```diff
@@ -0,0 +1,8 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <h1 id="title" class="title-class">Hello Microverse!</h1>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+</html>
```

Comment:
[OPTIONAL] The HTML consists of the head and body tag. Kindly add a `body` tag after the closing `head` tag on line 7 and move the `h1` into it. You can use this [link](https://github.com/jerryowusu/hello-microvese/blob/setup-the-project/index.html) as a guidance